tools/docker/sanity_check.Dockerfile

```diff
@@ -17,12 +17,12 @@ RUN touch /ok.txt
 # -------------------------------
 FROM python:3.6 as public-api-typed
 
-RUN pip install tensorflow-cpu==2.1.0
-RUN pip install typeguard==2.7.1
-RUN pip install git+https://github.com/gabrieldemarmiesse/typed_api.git@0.1.1
+RUN pip install tensorflow-cpu==2.1.0 \
+			typeguard==2.7.1 \
+			git+https://github.com/gabrieldemarmiesse/typed_api.git@0.1.1
 
 COPY ./ /addons
-RUN TF_ADDONS_NO_BUILD=1 pip install --no-deps -e /addons
+RUN TF_ADDONS_NO_BUILD=1 pip install --no-cache-dir --no-deps -e /addons
```

Comment:
When doing `pip install -e` it only creates a symlink, and since we use `--no-deps`, no other package is installed, so the cache won't be used in any case.

Reply:
Gotcha! Thanks @gabrieldemarmiesse.